### PR TITLE
Add speaker profiles to website

### PR DESCRIPTION
**Pull request by antstanley**
Add individual speaker pages with speaker data and talk details. Auto-generated from json in src/data

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,5 +32,8 @@
     "gulp-plumber": "^1.2.0",
     "gulp-uglify": "^3.0.0",
     "pump": "^3.0.0"
+  },
+  "dependencies": {
+    "showdown": "^1.8.6"
```
Comment (alexcasalboni) [inline package.json:37]: should be a dev dependency?

File: src/components/speaker-profile/buildHead.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+const buildHead = (slug, title, desc, previewImg) => {
+	
+	return `
+<head> 
+    <title>ServerlessDays London 2018</title>
```
Comment (alexcasalboni) [inline src/components/speaker-profile/buildHead.js:5]: This could be the speaker's name?

File: src/components/speaker-profile/buildHead.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
+const buildHead = (slug, title, desc, previewImg) => {
+	
+	return `
+<head>
```
Comment (alexcasalboni) [inline src/components/speaker-profile/buildHead.js:4]: `<html>` is missing!

Comment (antstanley) [reply src/components/speaker-profile/buildHead.js:4]: arrrgh
Thanks!

File: src/components/speaker-profile/buildHead.js (hunk 2 above)
Comment (antstanley) [reply src/components/speaker-profile/buildHead.js:5]: Yes! Doh!

File: package.json (hunk 1 above)
Comment (antstanley) [reply package.json:37]: effectively it doesn't matter, but for correctness yes ... just part of the build pipeline, only the html is shipped

**Pull request merged**